src/elan-cli/common.rs

```diff
@@ -204,6 +204,7 @@ pub fn update_all_channels(cfg: &Cfg, self_update: bool, force_update: bool) ->
 
     if toolchains.is_empty() {
         info!("no updatable toolchains installed");
+        info!("toolchains can be empty, try: elan toolchain install leanprover/lean4:nightly")
```

Comment:
note that elan can be used to install lean3 also, so I would not hard code a lean4 build in this message, I would recommend the message "no toolchains are installed, please try elan toolchain install instead."

Reply:
This is probably just an english grammar thing, but I don't know what "toolchains can be empty" means exactly, specifically the verb phase "can be" is future tense, it should just say the "toolchains are empty", but I still prefer "no toolchains are installed" as it is more clear, you don't have to guess what "empty" means, it means they are not "installed"....